data/pigui/game.lua

```diff
@@ -0,0 +1,512 @@
+						 ui.pointOnClock(center, reticuleCircleRadius - offset - width, hrs + 6),
+						 ui.theme.colors.navigationalElements, 1)
+	ui.addLine(ui.pointOnClock(center, reticuleCircleRadius - offset, hrs + 6),
+						 ui.pointOnClock(center, reticuleCircleRadius - offset, hrs + 6 - height_hrs),
+						 ui.theme.colors.navigationalElements, 1)
+	ui.addLine(ui.pointOnClock(center, reticuleCircleRadius - offset, 3),
+						 ui.pointOnClock(center, reticuleCircleRadius - offset + width/2, 3),
+						 ui.theme.colors.navigationalElements, 1)
+
+
+
+end
+
+local function displayReticuleCompass(center, heading)
+	local relevant = {}
+	local directions = { [0] = lc.COMPASS_N, [45] = lc.COMPASS_NE, [90] = lc.COMPASS_E, [135] = lc.COMPASS_SE, [180] = lc.COMPASS_S, [225] = lc.COMPASS_SW, [270] = lc.COMPASS_W, [315] = lc.COMPASS_NW }
+	local function cl(x)
+		if x < 0 then
+			return cl(x + 360)
+		elseif x >= 360 then
+			return cl(x - 360)
+		else
+			return x
+		end
+	end
+	local left = math.floor(heading - 45)
+	local right = left + 90
+	local d = left
+
+	local function stroke(d, p, n, height, thickness)
+		if d % n == 0 then
+			local a = ui.pointOnClock(center, reticuleCircleRadius, 2.8 * p - 1.4)
+			local b = ui.pointOnClock(center, reticuleCircleRadius + height, 2.8 * p - 1.4)
+			ui.addLine(a, b, ui.theme.colors.reticuleCircle, thickness)
+		end
+	end
+
+	ui.addLine(ui.pointOnClock(center, reticuleCircleRadius, 0),
+						 ui.pointOnClock(center, reticuleCircleRadius - 3, 0),
+						 ui.theme.colors.reticuleCircle, 1)
+
+	while true do
+		if d > right then
+			break
+		end
+		local p = (d - left) / 90
+		stroke(d, p, 15, 3, 1)
+		stroke(d, p, 45, 4, 1)
+		stroke(d, p, 90, 4, 2)
+		for k,v in pairs(directions) do
+			if cl(k) == cl(d) then
+				local a = ui.pointOnClock(center, reticuleCircleRadius + 8, 3 * p - 1.5)
+				ui.addStyledText(a, v, ui.theme.colors.navigationalElements, ui.fonts.pionillium.tiny, ui.anchor.center, ui.anchor.bottom, "")
+			end
+		end
+		d = d + 1
+	end
+end
+
+local function displayReticuleDeltaV(center)
+	-- ratio is 1.0 for full, 0.0 for empty
+	local function gauge(ratio, center, radius, color, thickness)
+		if ratio < 0 then
+			ratio = 0
+		end
+		if ratio > 0 and ratio < 0.001 then
+			ratio = 0.001
+		end
+		if ratio > 1 then
+			ratio = 1
+		end
+		ui.pathArcTo(center, radius + thickness / 2, ui.pi_2 + ui.pi_4, ui.pi_2 + ui.pi_4 + ui.pi_2 * ratio, 64)
+		ui.pathStroke(color, false, thickness)
+	end
+
+	local player = Game.player
+	local offset = 3
+	local thickness = 5
+
+	local deltav_max = player:GetMaxDeltaV()
+	local deltav_remaining = player:GetRemainingDeltaV()
+	local dvr = deltav_remaining / deltav_max
+	local deltav_maneuver = player:GetManeuverVelocity():magnitude()
+	local dvm = deltav_maneuver / deltav_max
+	local deltav_current = player:GetCurrentDeltaV()
+	local dvc = deltav_current / deltav_max
+
+	gauge(1.0, center, reticuleCircleRadius + offset, ui.theme.colors.deltaVTotal, thickness)
```

Comment:
Lots of similar looking code in this section. Perhaps refactor some of it in a variable ? That's more a note for future improvements, though.